lib/connect-backbone-to-react.js

```diff
@@ -0,0 +1,147 @@
+const hoistStatics = require('hoist-non-react-statics');
+const { Component, PropTypes, createElement } = require('react');
+const debounceFn = require('lodash.debounce');
+const isArray = require('lodash.isarray');
+
+function getDisplayName(name) {
+  return `connectBackboneToReact(${name})`;
+}
+
+function defaultMapModelsToProps(models) {
+  return Object.keys(models).reduce((acc, modelKey) => {
+    const model = models[modelKey];
+    acc[modelKey] = model.toJSON();
+    return acc;
+  }, {});
+}
+
+module.exports = function connectBackboneToReact(
+  mapModelsToProps,
+  options = {}
+) {
+  if (typeof mapModelsToProps !== 'function') {
+    mapModelsToProps = defaultMapModelsToProps;
+  }
+
+  const {
+    debounce = false,
+    events = {},
+    modelTypes = {},
+  } = options;
+
+  function getEventNames(modelName) {
+    let eventNames = events[modelName];
+
+    // Allow turning off event handlers by setting events to false.
+    if (eventNames === false) {
+      return [];
+    }
+
+    if (!isArray(eventNames)) {
+      eventNames = [];
+    }
+
+    if (eventNames.length === 0) {
```

Comment:
Very weird...I tried to remember my reasoning for this and I can't. Weird mind glitch. Nice catch.